src/sentry/integrations/slack/utils.py

```diff
@@ -257,11 +253,19 @@ def build_group_attachment(group, event=None, tags=None, identity=None, actions=
         },
     ]
 
-    fields = []
+    # If an event is specified, use the tags of the latest event (if one exists).
+    event_for_tags = event if event else group.get_latest_event()
 
-    if tags:
-        event_tags = event.tags if event else group.get_latest_event().tags
+    fallback_color = LEVEL_TO_COLOR["error"]
+    color = (
+        LEVEL_TO_COLOR.get(event_for_tags.get_tag("level"), fallback_color)
+        if event_for_tags
```

Comment:
Not sure if it it's possible to actually ever have a group that didn't have a latest event, but there was at least one existing test case that didn't, so I coded defensively here on L262 and L268.